examples/directive/myInterface.go

```diff
@@ -0,0 +1,84 @@
+package directive
+
+import (
+	"fmt"
+	"html/template"
+
+	//lint:ignore ST1001 This tests dot import support.
+	. "os"
+	renamed "text/template"
+
+	// Blank imports should not be included in mock files.
+	_ "embed"
+
+	"github.com/nicheinc/mock/examples/directive/internal"
+)
+
+// TODO: chan types
+// TODO: map types
```

Comment:
Are these still todos?

Reply:
I guess not? 1e410f8bee30e8b1b021766a2c462daf3352f69f

@nathanjcochran Do you remember if `map` and `chan` types as params/returns didn't use to generate correctly, or why these TODOs are here? Seven years later they seem to work no problem, but maybe I don't understand what those TODOs mean. :thinking:

Reply:
Looks like I added those TODOs while I was in the process of creating that test interface and working through some errors I was encountering at the time: https://github.com/nathanjcochran/mock/commit/3332ebb04e84a31eccffff452548c409f30dc3e7.

So I think it was just a reminder to myself to add those types to test interface, which I apparently never got around to doing. So I guess it was actually a reminder to you 😅.

Reply:
I think it was right around the time I discovered this Go bug: https://github.com/golang/go/issues/25262, and probably got sidetracked filing that issue.